### ISSUE TITLE
Latest Godot version 3.3.2

### ISSUE THREAD

**Issue by CompassDevs** [GhostWalker562/godot-admob-iOS-precompiled#11]
Hi Philip,

Last year you helped me out with your pre-compiled templates for godot 3.2.1 (with extra module ), i'm trying again using that old release but xcode always give me an error on build.

I have tried also to re-compile my self (latest godit 3.3.2) but i didn;t successed. must be very dumb :(.

Is a newer and better admob module i can use for iOS? and how can i also add share module?

Would you be so kind to assist me again?

Looking forward for your kind assistance!

**Comment by GhostWalker562** [comment]
Sure thing, where do you want to start? What is the issue your currently facing?

**Comment by CompassDevs** [comment]
Hello there,

Alright, at first i wanted to try to compile my own template, so i could add any module using always the latest version of Godot and latest Google SDK, i wasn;t able. Errors here and there which interrupts the creation. For the time being i'll keep it on hold.

My actual issue is, (i have managed to build in xcode, using the template you have made last year Godot 3.2.1 admob + share), that template is using old Google SDK (version 7.62) will it work if i update it with the latest? (8.8 is currently available), is also a requirement from Admob.

Does the template need to be recreated with the latest Google SDK? and then also linked in the xcode project?

I am also having a thread error on the first banner show, i didn;t spend much time on this i think is the code

**Comment by GhostWalker562** [comment]
Yes, if it's a requirement then you will have to recompile the template yourself with the latest SDK. You will have to look at the Godot Docs to figure that out.

The thread error is most likely on your end. You should only try to run one banner, and properly close it when it disappears.

**Comment by CompassDevs** [comment]
Will you be so kind to compile them for me?

**Comment by CompassDevs** [comment]
I think I'm asking too much 😬

**Comment by GhostWalker562** [comment]
At the moment, I can't because I'm on vacation. When I get back, I still can't help you because my MacBook is out of service and will not function anymore. That is a large reason behind why I can't support this repository at the moment.

**Comment by GhostWalker562** [comment]
I have written down a task to compile using GitHub actions, in the next month. Stay tuned, but I won't be able to start this project for a while. It would be better if you compiled it yourself if you want it promptly but I've found a course of action that will allow me to continue this repository.

**Comment by CompassDevs** [comment]
Thanks anyhow.

I hope this github idea of yours will come out soon. Meanwhile if you have
some tips on the compiling please share. I do follow the official
documentation, but suddenly I have errors and don't know how to solve.

Enjoy your vacation

**Comment by GhostWalker562** [comment]
I don't have tips besides follow the guides on the module itself. I won't be back until the 28th but I'll be sure to finish quick afterwards.

**Comment by GhostWalker562** [comment]
The module right now is outdated. I tried to use actions but it didn't work. I'm getting a Mac for college, but won't get it until September. Check out this plugin, https://github.com/Poing-Studios/godot-admob-ios . Maybe this will give you better luck. Sorry about the inconvenience!

**Comment by CompassDevs** [comment]
Hi, thanks for getting back to me first of all.

I also thought that module is too old.

Now, from my side I have managed to compile godot 3.3.2 without any  extra
module. So that problem is solved. I just need an admob and share module
that works with the 3.3.2.

Do you think the admob you suggested will? And about the share ?

**Comment by GhostWalker562** [comment]
Interesting, I wasn't able to compile it at all but I'll see again. I'd say use the plugin I sent you and recompile with the share module. The process of recompiling with the share module is the exact same as the admob one.

**Comment by CompassDevs** [comment]
I had to update scons and xcode. Then it worked.

Now, when I try to compile including the share it gets an error and doesn't
complete the compile

**Comment by GhostWalker562** [comment]
I'd advise just restart the process and try your best at it again. I apologize but currently I can't help any further due to hardware limitations.
